plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -0,0 +1,191 @@
+import welcomeScreenVideo from "@web/extensions/sidebar/core/assets/welcomeScreenVideo.png";
+// import Backdrop from "@web/extensions/sidebar/modals/welcomescreen/component/Backdrop";
+import useHooks from "@web/extensions/sidebar/modals/welcomescreen/hooks";
+import { Checkbox, Icon } from "@web/sharedComponents";
+import Video from "@web/sharedComponents/Video";
+import { styled } from "@web/theme";
+
+const WelcomeScreen: React.FC = () => {
+  const { ShowVideo, handleDontShowAgain, dontShowAgain, handleShowVideo, handleClose } =
+    useHooks();
+
+  return (
+    <Wrapper>
+      <CloseButton>
+        <Icon size={32} icon="close" onClick={handleClose} />
+      </CloseButton>
+      {!ShowVideo ? (
+        <>
+          <InnerWrapper>
+            <TryMapWrapper>
+              <TextWrapper width={192} height={46}>
+                <Text weight={700} size={48}>
+                  ようこそ
+                </Text>
+              </TextWrapper>
+              <ImgSection>
+                <TextWrapper width={215} height={32}>
+                  <Text weight={500} size={20}>
+                    マップを使ってみる
+                  </Text>
+                </TextWrapper>
+                <ImgWrapper>
+                  <img src={welcomeScreenVideo} onClick={handleShowVideo} />
+                </ImgWrapper>
+              </ImgSection>
+            </TryMapWrapper>
+            <BtnsWrapper>
+              <ButtonWrapper>
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+              <ButtonWrapper>
+                <Icon size={20} icon="plusCircle" color="#fafafa" />
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+            </BtnsWrapper>
+          </InnerWrapper>
+          <CheckWrapper>
+            <Checkbox checked={dontShowAgain} onClick={handleDontShowAgain}>
+              <TextWrapper width={192} height={46}>
+                <Text weight={700} size={14}>
+                  閉じて今後は表示しない
+                </Text>
+              </TextWrapper>
+            </Checkbox>
+          </CheckWrapper>
+        </>
+      ) : (
+        <VideoWrapper>
+          <Video width=" 1142" height="543" src="https://www.youtube.com/embed/pY2dM-eG5mA" />
+        </VideoWrapper>
+      )}
+      {/* <Backdrop onClose={handleClose} /> */}
+    </Wrapper>
+  );
+};
+export default WelcomeScreen;
+
+const Wrapper = styled.div``;
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  padding: 0px;
+  gap: 119px;
+  position: absolute;
+  width: 742px;
+  height: 316px;
+  left: calc(50% - 742px / 2 + 0.5px);
+  top: calc(50% - 316px / 2 - 30.5px);
+`;
```

Comment:
These styles need to be updated.
You set the width and height in the show API options so Wrapper needs to be 100% width and height, and then here would be where you set the background color, etc. Something like:
```
const Wrapper = styled.div`
   height: 100%;
   width: 100%;
   display: flex;
   justify-content: center;
  align-items: center;
  background: whateverTheSemiTransparentColorIs;
...
...
`;
```
The above will give you the full screen overlay and position the InnerWrapper (where the two buttons, the video and the checkbox will be) in the center of the screen.
With that, you can simplify InnerWrapper's CSS as you don't need to position absolute it, give the gap, etc

Reply:
Also, it seems like a lot of the styled components' CSS is written assuming they need to position absolutely, which with the above changes you shouldn't need to do so much work.

Reply:
Done

Reply:
e8d02a5faeea4dc4732056ccaf88ca6f433e776c